### ISSUE TITLE
Note ID prevents proper nesting of lists.

### ISSUE THREAD

**Issue by MahmoudFawzyKhalil** [Pseudonium/Obsidian_to_Anki#173]
![image](https://user-images.githubusercontent.com/73137611/102529096-1a9b4980-40a8-11eb-802c-0b42432c87ec.png)
![image](https://user-images.githubusercontent.com/73137611/102529105-1cfda380-40a8-11eb-8289-c665e7df2d28.png)

Putting a line break fixes it

![image](https://user-images.githubusercontent.com/73137611/102529164-343c9100-40a8-11eb-9e93-1e2b9742bdca.png)
![image](https://user-images.githubusercontent.com/73137611/102529169-36065480-40a8-11eb-9f02-75f7e5a7337a.png)

But running scan vault again resets it and removes the line break

**Comment by Pseudonium** [comment]
Right, I might be able to do a special case for this. At the moment, the script automatically replaces ‘double new line then ID’ by ‘single beeline then ID’, since it looks nicer usually.

**Comment by MahmoudFawzyKhalil** [comment]
Thank you for your prompt reply. I look forward to a solution or some sort of work around. 

BTW this addon/plugin is incredible. Thank you for your hard work!

**Comment by Pseudonium** [comment]
Hmm, I've tested this with some online markdown editors, and this doesn't seem to break nested lists. I wonder if this could be a bug in Obsidian's markdown to HTML engine? I'll file this as a bug report there for now.

**Comment by Pseudonium** [comment]
Alright, here's the [bug report on the Obsidian forum](https://forum.obsidian.md/t/nested-list-rendering-broken-if-line-after-is-not-empty/10133)

**Comment by RoamanEmpire** [comment]
I'm also experiencing this and it is pretty annoying since I like to use lists + folding to get some extra "active recall".

Currently I've modified the plugin code locally to add two new line lines before the ID, although it is a bit ugly.

@Pseudonium as a temporary fix, do you think it would be possible to (have the option to) move the ID to the top of a note instead of the bottom. Example:
<img width="431" alt="image" src="https://user-images.githubusercontent.com/69714537/113153653-9a07ef80-9237-11eb-9151-442d73c19f6c.png">

Alternatively, since I'm still unfamiliar with the codebase and typescript, could you point me to the section of the code where this change can be implemented? Thank you!

**Comment by ShootingKing-AM** [comment]
> Alright, here's the [bug report on the Obsidian forum](https://forum.obsidian.md/t/nested-list-rendering-broken-if-line-after-is-not-empty/10133)

This bug report maked as wont-fix on obsidian side. Therefore, we ourselves have to comeup with a fix.
